wasp/boards/manifest_240x240.py

```diff
@@ -17,6 +17,7 @@
     'apps/stopwatch.py',
     'apps/fibonacci_clock.py',
     'apps/snake.py',
+    'apps/calc.py',
```

Comment:
I'm afraid I'm not happy to add an app with a 15K image embedded in it into the default flash image.

IMHO the calculator should be drawn by having a single blank black and white button that is drawn multiple times (using things like fg=0xf800 to draw the button in the different colours). The button labels can then be drawn using the font renderer rather than included as part of one massive image.